src/lib/plugin.js

```diff
@@ -56,12 +56,22 @@ class FiveBellsLedger extends EventEmitter2 {
       throw new TypeError('Expected an options object, received: ' + typeof options)
     }
 
+    if (typeof options.prefix !== 'string') {
+      throw new TypeError('Expected options.prefix to be a string, received: ' +
+        typeof options.prefix)
+    }
+
+    if (options.prefix.slice(-1) !== '.') {
+      throw new Error('Expected options.prefix to end with "."')
+    }
+
     if (typeof options.auth !== 'object') {
       throw new TypeError('Expected options.auth to be an object, received: ' +
         typeof options.auth)
     }
 
-    this.id = options.id || null
+    this.prefix = options.prefix
```

Comment:
I think that `this.prefix` should be passed in through `options.auth` instead of just `options`, because the way that the prefix is configured is ledger-specific.